specs/opcode/5BJUMPDEST.md

```diff
@@ -0,0 +1,40 @@
+# JUMPDEST op code
+## Procedure
+   JUMPDEST is a special opcode which only marks an address that can be jumped to.   in other words it is an metadata to annotate possible jump destinations.  
+
+   For example,  blow op code sequences, JUMPDEST marks '004D' can be jumped to, if there is no JUMPDEST annotation, Jumping to '004D' will fail as jump op code will require JUMPDEST as valid destination.  
+
+```
+      004D    5B  JUMPDEST  
+      004E    34  CALLVALUE  
+      004F    80  DUP1  
+      0050    15  ISZERO  
+      0051    60  PUSH1 0x58  
+      0053    57  *JUMPI  
+      0054    60  PUSH1 0x00  
+      0056    80  DUP1  
+      0057    FD  *REVERT  
+      0058    5B  JUMPDEST  
+      0059    50  POP  
+      005A    60  PUSH1 0x62  
+      005C    60  PUSH1 0x04  
+      005E    35  CALLDATALOAD  
+      005F    60  PUSH1 0x86  
+      0061    56  *JUMP"  
```

Comment:
illustrating this helps understand jumpdest apply,  since interpreter treat jumpdest  just as annotation, no more handling when executing op code other than state transitions .    like to hear if you think more